core/src/main/scala/pureconfig/package.scala

```diff
@@ -4,25 +4,47 @@
 /**
  * @author Mario Pastorelli
  */
+
 import java.io.{ OutputStream, OutputStreamWriter }
 import java.nio.charset.StandardCharsets.UTF_8
 import java.nio.file.{ Files, Path }
 
 import scala.reflect.ClassTag
 
 import com.typesafe.config.{ Config => TypesafeConfig, _ }
-import pureconfig.ConvertHelpers._
 import pureconfig.backend.ConfigFactoryWrapper._
 import pureconfig.backend.PathUtil._
 import pureconfig.error._
 
 package object pureconfig {
 
+  /**
+   * Object containing useful constructors and utility methods for `ReaderResult`s.
+   */
+  object ReaderResult {
```

Comment:
Shouldn't this also be moved to `ConfigReader.Result`?

Reply:
👍 cb45d1f